recipes/sdl/all/conanfile.py

```diff
@@ -183,13 +179,7 @@ def package_id(self):
             del self.info.options.sdl2main
 
     def build_requirements(self):
-        if self.settings.os == "Macos" and cross_building(self):
-            # Workaround for CMake bug with error message:
-            # Attempting to use @rpath without CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG being
-            # set. This could be because you are using a Mac OS X version less than 10.5
-            # or because CMake's platform configuration is corrupt.
-            # FIXME: Remove once CMake on macOS/M1 CI runners is upgraded.
-            self.tool_requires("cmake/3.27.9")
+        self.tool_requires("cmake/[>3.27 <4]")
```

Comment:
This error still persists? We have update Mac machines to M2, maybe we have a different result now.

Reply:
somehow it failed without this tool_requires: https://c3i.jfrog.io/artifactory/misc-v2/logs/pr/22246/8-macos-clang/sdl/2.28.5/cde1046a936a59525b593ea75621a3b236a649fa-build.txt